robot/rospackages/src/image_scp/image_scp.py

```diff
@@ -0,0 +1,78 @@
+import fnmatch
+import os
+from paramiko import SSHClient
+from scp import SCPClient, SCPException
+import argparse
+from config import (host, user, remote_path_1)
+"""
+Script takes in the image as a command line argument.
+SSH keys need to be configured on both remote and local machine in order for the script to work.
+Host name, user nmae, and remote path are all imported from a config file. Update the information
+there so that this script is portable .
+
+"""
+
+class remote_connection:
+
+    def __init__(self, host, user, remote_path, file_name):
+        self.host = host
+        self.user = user
+        self.remote_path = remote_path
+        self.file_name = file_name
+#initiate the connection to the host machine
+    def connect(self):
+        self.client = SSHClient()
+        self.client.load_system_host_keys()
+        self.client.connect(hostname=self.host, username=self.user)
+        self.scp = SCPClient(self.client.get_transport())
+
+    def disconnect(self):
+        self.client.close()
+        self.scp.close()
+
+    def send_image(self, image):
+        try:
+            self.scp.put(image, recursive=True, remote_path=self.remote_path)
+        except SCPException as error:
+            raise error
+
+    def send_batch_images(self, image_directory):
+        bulk_tranfer = [self.send_image(image) for image in image_directory]
+        print(len(bulk_transfer) + "Images have been transfered")
+
+
+if __name__ == "__main__":
+
+  #takes the image as a command line argument
+    parser = argparse.ArgumentParser(description="the file that gets sent")
+    parser.add_argument("image_directory", action="append")
+    args = parser.parse_args()
+    to_transfer = args.image_directory[0]
+
+    image_files = []
+
+    #Takes in a directory name from which a config file and jpeg files are extracted
+    for name in os.listdir('test-file/'):
```

Comment:
is this `test-file` meant to be used in production or is this a mistake?

Reply:
No, not meant for production. It's a mistake left over from testing. I'm  addressing your suggestions now.

Reply:
@MartensCedric  Fixed this issue and debugged the rest of the code. It "Works on my machine" now.